src/main.rs

```diff
@@ -185,7 +185,7 @@ impl<'a> Generator<'a> {
             .collect();
         let data = Releases {
             releases: releases,
-            feed_updated: chrono::Utc::now().to_rfc3339(),
+            feed_updated: chrono::Utc::now().with_nanosecond(0).unwrap().to_rfc3339(),
```

Comment:
Maybe there's no need to change this, but I thought it was OK to keep them aligned.